ganttproject/src/net/sourceforge/ganttproject/GanttOptions.java

```diff
@@ -55,7 +55,12 @@
 import javax.xml.transform.sax.TransformerHandler;
 import javax.xml.transform.stream.StreamResult;
 import java.awt.*;
-import java.io.*;
+import java.io.BufferedOutputStream;
```

Comment:
Please revert changes in this file. Formatting changes just spoil the pull request and make it incomprehensible when mixed with other meaningful changes.